modules/customizing-the-appearance/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -1,129 +1,60 @@
-          title: My Plugin Page # optional, same as `menuItem.text` in `dynamicRoutes` <3>
-          priority: 10 # optional, defines the order of menu items in the sidebar <4>
-          parent: favorites # optional, defines parent-child relationships for nested menu items <5>
-----
-
-You can modify the fields in the previous example to configure the desired order and parent-child relationships of the sidebar menu items.
-
-<1> This attribute represents a unique name in the main sidebar navigation. It can denote either a standalone menu item or a parent menu item. If this attribute represents a plugin menu item, the name of the attribute must match with the corresponding path in `dynamicRoutes`. For example, if `dynamicRoutes` defines `path: /my-plugin`, then `menu_item_name` must be defined as `my-plugin`.
+    <package_name>: # <1>
+      menuItems: # <2>
+        <menu_item_name>: # <3>
+          icon: # home | group | category | extension | school | _<your_icon>_ # <4>
+          title: _<plugin_page_title>_ # <5>
+          priority: 10 # <6>
+          parent: favorites # <7>
+----
+<1> `_<package_name>_`: Specifies the name of the package. This name is the same as the `scalprum.name` key in the plugin `package.json`.
+<2> `menuItems`: (Optional) Use to configure plugin menu items in the main sidebar navigation.
+<3> `_<menu_item_name>_`: Specifies a unique name in the main sidebar navigation for either a standalone menu item or a parent menu item. If this field specifies a plugin menu item, the name of the menu item must match the name using in the corresponding path in `dynamicRoutes`. For example, if `dynamicRoutes` defines `path: /my-plugin`, then `menu_item_name` must be defined as `my-plugin`.
+<4> `icon`: (Optional) Specifies the icon for the menu item. You can use default icons, such as: `home`, `group`, `category`, `extension`, and `school`. You can also use an SVG or HTML image icon, such as: `icon: <svg width="20px" height="20px" viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg" fill="#ffffff">...</svg>`  or `icon: https://img.icons8.com/ios-glyphs/20/FFFFFF/shop.png`.
+<5> `title`: (Optional) Specifies the title of the menu item. Omit it when the title is already specified in the `dynamicRoutes` configuration under `menuItem.text`.
+<6> `priority`: (Optional) Sets the order in which menu items appear in the sidebar. The default priority is 0, which places the item at the bottom of the list. A higher priority value places the item higher in the sidebar. You can define this field for each section.
+<7> `parent`: (Optional) Specifies the parent menu item under which the current item is nested. If this field is used, the parent menu item must be defined elsewhere in the `menuItems` configuration of any enabled plugin. You can define this field for each section.
+
+. To add *Main menu item* to the dynamic plugins configuration file, add the `default.` prefix to its key in your `{my-app-config-file}` file:
```

Comment:
Exact reference in the style guide:
> If an element is not labeled in the GUI, refer to the element by a generic description and do not use bold text. For example, if a search field is not labeled in the GUI, write it as "the search field", not "the Search field".
https://redhat-documentation.github.io/supplementary-style-guide/#user-interface-elements 

```suggestion
. To add a main menu item to the dynamic plugins configuration file, add the `default.` prefix to its key in your `{my-app-config-file}` file:
```